compiler/resolution/postFold.cpp

```diff
@@ -410,7 +417,14 @@ static Expr* postFoldPrimop(CallExpr* call) {
 
         INT_ASSERT(ie && ie->symbol()->isParameter() && get_int(ie, &val));
```

Comment:
Ugh, `get_int` is computing something that is used later in the function in the control flow, but it's being called from an `INT_ASSERT`. That's bad because one might reasonably assume that the `INT_ASSERT` can be removed at only the cost of less error checking.

Could you move the `get_int` call to a separate line?

Reply:
> Could you move the `get_int` call to a separate line?

Done.